docs/MODULE_BOUNDARIES.md

```
@@ -0,0 +1,1631 @@
+    WebhookService --> Types
+    WebhookService --> Middleware
+    WorkflowService --> Types
+    BackgroundWorker --> Types
+    BackgroundWorker --> Utils
+    GitService --> Types
+    GitService --> Contracts
+    MCPServer --> Types
+    MCPServer --> Utils
+
+    %% Platforms to Shared Libraries
+    Web --> Components
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)